src/django/api/serializers/v1/production_locations_serializer.py

```diff
@@ -45,6 +47,15 @@ class ProductionLocationsSerializer(Serializer):
         choices=['asc', 'desc'],
         required=False
     )
+    aggregation = ChoiceField(
+        choices=['hexgrid'],
+        required=False,
+    )
+    precision = IntegerField(
```

Comment:
I would suggest re-naming this parameter into `geohex_grid_precision`, that would indicate what this parameter relates to and make the API easier to understand to the end user.

Reply:
Done.